abcclassroom/quickstart.py

```diff
@@ -0,0 +1,93 @@
+"""
+abc-classroom.dir-setup
+========================
+
+"""
+
+import os
+import shutil
+import argparse
+import abcclassroom
```

Comment:
@nkorinek just curious - why are you importing everything here? i'd rather see just the things this module needs imported here.

Reply:
The module does use all of those libraries, is there somewhere better to import them? @lwasser

Reply:
great question. Does it use ALL of the functions in all modules? for example i do'nt think it uses ok or any of the git  / github stuff. 

so i would suggest looking at the notebook module. notice how tim imports just the things he needs like this:

`from .utils import chdir`

so just import the methods that you need from the specific modules. when you import abcclassroom you are importing absolutely everything. so to be clear i'm referring to you using

`import abcclassroom` vs 
`from .utils import chdir` <- example ot just importing the functions that you USE in a particular module rather than the entire thing. 

does that make sense? i may be missing something as well :)

Reply:
@lwasser ya I can definitely get more specific with what I import!